charts/gitlab-ci-pipelines-exporter/Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: redis
-  repository: https://charts.bitnami.com/bitnami
-  version: 16.4.0
-digest: sha256:ccba2e0d5b12df3a803d7c0a5bf797600038a80bc3d7cd9b9c72a2ae54a6473d
-generated: "2022-02-11T14:00:44.981418+01:00"
+  repository: file://./deps/bitnami/redis/
```

Comment:
```suggestion
  repository: https://raw.githubusercontent.com/bitnami/charts/eb5f9a9513d987b519f0ecd732e7031241c50328/bitnami```

Tarballs remains untouched. How about using old index by specifying raw context?

Reply:
That would work just fine, and is actually part of [bitnami's suggested workaround](https://github.com/bitnami/charts/issues/10539).  I didn't initially suggest that because it seems a bit flimsy to tie to a semi-random commit of a branch that they're keeping around for legacy, but it would require fewer changes.

Reply:
Going with your suggestion, thanks @czomo!